deployment/kustomizations/base/deploy-wren-ai-service.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wren-ai-service-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: wren-ai-service
+  template:
+    metadata:
+      labels:
+        app: wren-ai-service
+    spec:
+      containers:
+        - name: wren-ai-service
+          image: ghcr.io/canner/wren-ai-service:0.3.7
+          env:
+          - name: WREN_AI_SERVICE_PORT
+            valueFrom: 
+              configMapKeyRef:
+                name: wren-config
+                key: WREN_AI_SERVICE_PORT
+          - name: OPENAI_BASE_URL
+            valueFrom: 
+              secretKeyRef:
+                name: wrenai-secrets
+                key: OPENAI_BASE_URL
```

Comment:
This is also unimplemented. Would be nice if we remove this.

Reply:
as you say